smart-send-logistics/build/smart-send-block-frontend.js

```diff
@@ -0,0 +1,667 @@
+ * External dependencies
+ */
+
+
+
+
+
+
+/**
+ * Internal dependencies
+ */
+
+
+const Block = ({
+  checkoutExtensionData,
+  extensions
+}) => {
+  const isCalculating = (0,_wordpress_data__WEBPACK_IMPORTED_MODULE_4__.useSelect)(select => {
+    const store = select("wc/store/checkout");
+    return store.isCalculating();
+  });
+  const {
+    setExtensionData
+  } = checkoutExtensionData;
+  const debouncedSetExtensionData = (0,_wordpress_element__WEBPACK_IMPORTED_MODULE_1__.useCallback)((0,lodash__WEBPACK_IMPORTED_MODULE_5__.debounce)((namespace, key, value) => {
+    setExtensionData(namespace, key, value);
+  }, 1000), [setExtensionData]);
+  const validationErrorId = "smart-send-other-value";
+  const {
+    setValidationErrors,
+    clearValidationError
+  } = (0,_wordpress_data__WEBPACK_IMPORTED_MODULE_4__.useDispatch)("wc/store/validation");
+  const validationError = (0,_wordpress_data__WEBPACK_IMPORTED_MODULE_4__.useSelect)(select => {
+    const store = select("wc/store/validation");
+    return store.getValidationError(validationErrorId);
+  });
+  const [selectedPickupPoint, setselectedPickupPoint] = (0,_wordpress_element__WEBPACK_IMPORTED_MODULE_1__.useState)("try-again");
+  (0,_wordpress_element__WEBPACK_IMPORTED_MODULE_1__.useEffect)(() => {
+    var street = setValue("shipping-address_1");
+    var city = setValue("shipping-city");
+    var country = setValue("components-form-token-input-0");
+    country = _countries__WEBPACK_IMPORTED_MODULE_7__.countries[country];
+    var postcode = setValue("shipping-postcode");
+    var selected = jQuery(".wc-block-components-shipping-rates-control").find("input[type=radio]:checked").val();
+    if (isCalculating) {
+      if (postcode !== null && street !== null && city !== null && country !== null) {
+        findClosestAgentByAddress(selected, country, postcode, city, street);
+      }
+    }
+  }, [isCalculating]);
+  (0,_wordpress_element__WEBPACK_IMPORTED_MODULE_1__.useEffect)(() => {
+    setExtensionData("smart-send-logistics", "selectedPickupPoint", selectedPickupPoint);
+  }, [setExtensionData, selectedPickupPoint]);
+  const [hasInteractedWithOtherInput, setHasInteractedWithOtherInput] = (0,_wordpress_element__WEBPACK_IMPORTED_MODULE_1__.useState)(false);
+  (0,_wordpress_element__WEBPACK_IMPORTED_MODULE_1__.useEffect)(() => {
+    if (setselectedPickupPoint !== "other") {
+      if (validationError) {
+        clearValidationError(validationErrorId);
+      }
+      return;
+    }
+    setValidationErrors({
+      [validationErrorId]: {
+        message: (0,_wordpress_i18n__WEBPACK_IMPORTED_MODULE_3__.__)("Please add some text", "smart-send-logistics"),
+        hidden: !hasInteractedWithOtherInput
+      }
+    });
+  }, [clearValidationError, setselectedPickupPoint, setValidationErrors, validationErrorId, debouncedSetExtensionData, validationError]);
+  return (0,react__WEBPACK_IMPORTED_MODULE_0__.createElement)("div", {
+    className: "wp-block-smart-send-not-at-home"
```

Comment:
Why is there a class name `not-at-home`? That soulds like a left over from the tutorial.

Reply:
updated.